inspirehep/modules/orcid/api.py

```diff
@@ -50,12 +51,19 @@ def push_record_with_orcid(recid, orcid, oauth_token, put_code=None):
         oauth_token (string): ORCID user OAUTH token
         put_code (Union[string, NoneType]): put-code to push record onto,
             if None will push as a new record
+        old_hash (Union[string, NoneType]): previous hash of the record
```

Comment:
Nit: this can be `Optional[str]`, and it should be `str` instead of `string` everywhere.

Reply:
Why `str`? `string` iirc is different and means `Union[str, unicode]` (and equivalent in python3).

Reply:
:+1: didn't know that